background/providers/kat.js

```diff
@@ -0,0 +1,103 @@
+var https = require('https');
+var zlib = require('zlib');
+var path = require('path');
+var nconf = require('nconf');
+var mongoose = require('mongoose');
+var Category = require('../../app/models/Category.js');
+var Torrent = require('../../app/models/Torrent.js');
+
+nconf.use('memory');
+nconf.argv().env('__').file({
+    file: path.resolve(__dirname + '/../../config.json')
```

Comment:
Same goes for here and really we should look at adding nconf to the app global so we don't have to re-init it.